language/move-compiler/src/parser/lexer.rs

```diff
@@ -402,6 +414,34 @@ impl<'input> Lexer<'input> {
         self.token = token;
         self.cur_end = self.cur_start + len
     }
+
+    /// Construct the text info for every token in the lexer by advancing until EOF
```

Comment:
It seems like this is used to parse a single line - is looking for EOF as the end of iteration a correct choice here(@tnowacki)?

Reply:
@tnowacki a friendly reminder for this^